doc/nrf/libraries/networking/lwm2m_server_setup.rst

```diff
@@ -0,0 +1,95 @@
+.. _server_setup_lwm2m:
+
+Server setup
```

Comment:
**suggestion:** split up the sections so the instructions are grouped **per server**. This will allow users to quickly skip the instructions for the server they are not using.